### PR TITLE
Take 0.3.2 since that's latest release

### PR DESCRIPTION
**Pull request by petemounce**

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by johnynek** [comment]
can you update the tests:

https://github.com/bazelbuild/rules_scala/blob/master/.travis.yml#L19

**Comment by petemounce** [comment]
Sure, done.

File: .travis.yml [hunk 1]
```diff
@@ -16,10 +16,10 @@ jdk:
   - oraclejdk8
 
 before_install:
-  - wget 'https://github.com/bazelbuild/bazel/releases/download/0.3.1/bazel-0.3.1-installer-linux-x86_64.sh'
+  - wget 'https://github.com/bazelbuild/bazel/releases/download/0.3.2/bazel-0.3.2-installer-linux-x86_64.sh'
   - sha256sum -c .bazel-installer-linux-x86_64.sh.sha256
```
Comment (johnynek) [inline .travis.yml]: you're going to have to update the sha file, or this check will fail.

Comment (petemounce) [inline .travis.yml]: How is it produced, `shasum 1 <file> > .bazel-installer-linux-x86_64.sh.sha256`?

Comment (johnynek) [inline .travis.yml]: download the file here: https://github.com/bazelbuild/bazel/releases

Comment (petemounce) [inline .travis.yml]: Ah; thanks. Done.

**Comment by johnynek** [comment]
@petemounce I'm happy to merge, but can you confirm that you have signed the CLA again? Sorry for the bother.

**Comment by smparkes** [comment]
FWIW, 0.4.0 is out now (and borked a bunch of our stuff, though haven't looked to see if it was the scala rules; sigh).

Also, Scala 2.12 is out too.

Just saying.

**Comment by johnynek** [comment]
Yeah we need to look at both of those issues.

**Pull request merged**